assertion/assertion.mbt

```diff
@@ -69,3 +69,51 @@ test "assert_eq.eq" {
   assert_eq(1, 1)?
   assert_eq("123", "123")?
 }
+
+pub fn assert_is[T : Debug](a : T, b : T) -> Result[Unit, String] {
+  if a === b {
+    Ok(())
+  } else {
+    let a = debug_string(a)
+    let b = debug_string(b)
+    Err("assertion failed for `\(a) === \(b)`")
+  }
```

Comment:
This is preferably `@option.unless(a === b, fn() { ... })`, but crippled by the lack of `test_import` in `moon.pkg.json`, and/or a better handling of cyclic dependencies.

Reply:
`test_import` will be added as soon as next week